{
  "stats": {
    "suites": 0,
    "tests": 1,
    "passes": 0,
    "pending": 0,
    "failures": 1,
    "start": "2025-08-05T11:45:44.529Z",
    "end": "2025-08-05T11:45:45.125Z",
    "duration": 596,
    "testsRegistered": 1,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "c8e5ebdf-30ea-48ff-9114-a0e0301ca5f4",
      "title": "",
      "fullFile": "cypress\\e2e\\tests\\loginTest.cy.js",
      "file": "cypress\\e2e\\tests\\loginTest.cy.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [
        {
          "title": "An uncaught error was detected outside of a test",
          "fullTitle": "An uncaught error was detected outside of a test",
          "timedOut": null,
          "duration": 523,
          "state": "failed",
          "speed": null,
          "pass": false,
          "fail": true,
          "pending": false,
          "context": null,
          "code": "throw err;",
          "err": {
            "message": "CypressError: The following error originated from your test code, not from Cypress.\n\n  > `Cypress.Commands.add()` is used to create new commands, but `wait` is an existing Cypress command.\n\nPlease use `Cypress.Commands.overwrite()` if you would like to overwrite an existing command.\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n\nhttps://on.cypress.io/custom-commands",
            "estack": "CypressError: The following error originated from your test code, not from Cypress.\n\n  > `Cypress.Commands.add()` is used to create new commands, but `wait` is an existing Cypress command.\n\nPlease use `Cypress.Commands.overwrite()` if you would like to overwrite an existing command.\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n\nhttps://on.cypress.io/custom-commands\n    at ./cypress/support/commands.js (webpack://ecom_aut/./cypress/support/commands.js:9:17)\n    at __webpack_require__ (webpack://ecom_aut/webpack/bootstrap:19:0)\n    at eval (webpack://ecom_aut/./cypress/support/e2e.js:17:0)\n    at eval (http://localhost:60055/__cypress/tests?p=cypress\\support\\e2e.js:539:3)\n    at eval (http://localhost:60055/__cypress/tests?p=cypress\\support\\e2e.js:541:12)\n    at eval (<anonymous>)\n\nFrom Cypress Internals:\n      at internalError (http://localhost:60055/__cypress/runner/cypress_runner.js:111969:91)\n      at Object.add (http://localhost:60055/__cypress/runner/cypress_runner.js:112016:11)\n      at <unknown> (http://localhost:60055/__cypress/runner/cypress_runner.js:111853:23)\n      at tryCatcher (http://localhost:60055/__cypress/runner/cypress_runner.js:1807:23)\n      at Object.gotValue (http://localhost:60055/__cypress/runner/cypress_runner.js:6476:18)\n      at Object.gotAccum (http://localhost:60055/__cypress/runner/cypress_runner.js:6465:25)\n      at Object.tryCatcher (http://localhost:60055/__cypress/runner/cypress_runner.js:1807:23)\n      at Promise._settlePromiseFromHandler (http://localhost:60055/__cypress/runner/cypress_runner.js:1519:31)\n      at Promise._settlePromise (http://localhost:60055/__cypress/runner/cypress_runner.js:1576:18)\n      at Promise._settlePromiseCtx (http://localhost:60055/__cypress/runner/cypress_runner.js:1613:10)\n      at _drainQueueStep (http://localhost:60055/__cypress/runner/cypress_runner.js:2411:12)\n      at _drainQueue (http://localhost:60055/__cypress/runner/cypress_runner.js:2400:9)\n      at Async._drainQueues (http://localhost:60055/__cypress/runner/cypress_runner.js:2416:5)\n      at Async.drainQueues (http://localhost:60055/__cypress/runner/cypress_runner.js:2286:14)",
            "diff": null
          },
          "uuid": "a216fc7b-5999-48d2-bee9-03b15531093f",
          "parentUUID": "c8e5ebdf-30ea-48ff-9114-a0e0301ca5f4",
          "isHook": false,
          "skipped": false
        }
      ],
      "suites": [],
      "passes": [],
      "failures": [
        "a216fc7b-5999-48d2-bee9-03b15531093f"
      ],
      "pending": [],
      "skipped": [],
      "duration": 523,
      "root": true,
      "rootEmpty": false,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": true,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "7.1.3"
    },
    "marge": {
      "options": {
        "reportDir": "cypress/reports",
        "overwrite": false,
        "html": true,
        "json": true
      },
      "version": "6.2.0"
    }
  }
}